pages.es/common/git-remote.md

```diff
@@ -0,0 +1,24 @@
+# git remote
+
+> Gestiona el conjunto de repositorios rastreados ("remotos").
+> Más información: <https://git-scm.com/docs/git-remote>.
+
+- Muestra una lista de los remotos existentes, sus nombres y URL:
+
+`git remote -v`
+
+- Añade un remoto:
+
+`git remote add {{remote_name}} {{remote_url}}`
```

Comment:
Perhaps the bits in `{{}}` could be translated too? Same goes the  other examples.

Reply:
Of course! I forgot that. I will translate it and commit in a few minutes. Thanks for point this detail @sbrl!

Reply:
@sbrl, done!